docs/source/settings.rst

```diff
@@ -31,6 +31,26 @@ Whether to enable Spidermon.
 SPIDERMON_EXPRESSIONS_MONITOR_CLASS
 -----------------------------------
 
+
+Default: ``spidermon.python.monitors.ExpressionMonitor``
+
+A fully qualified path for a Python Class that should inherit from 
+``spidermon.python.monitors.ExpressionMonitor``.
+
+This class will be used to generate on-the-fly monitors for the folowing settings:
```

Comment:
I would just go with:

```suggestion
This class will be used to generate :ref:`expression monitors <expression-monitors>`.
```

And remove the remaining text below (list and paragraph after the list). Users interested in expression monitors can follow the link and find everything they need.